### PR TITLE
tango_mvp inference

### PR DESCRIPTION
**Pull request by genshimamber**

### CODE REVIEW

**Review by wtomin — APPROVED** [review]
Tested the inference example. The result sounds ok to me.

File: examples/tango/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+IPython
```
[the remark this answers is not in the document]
Comment (genshimamber) [reply examples/tango/requirements.txt:1]: ok I'll add those in next pr

File: examples/tango/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+"a dog is barking" [audio](https://github.com/genshimamber/mindone/assets/145047261/f67cf926-d0e7-43b4-a690-2f05eb7a96db)
+
+"a cat is meowing" [audio](https://github.com/genshimamber/mindone/assets/145047261/647d7834-5ff5-4e70-96a7-d9ae718a0285)
+
+Note: we manually converted the generated recordings from `.wav` to `.webm` so we could put them in readme.
+
+## Getting Started
+
+1. download weights from [tango_full_ft_audiocaps-fa8f707f](https://download.mindspore.cn/toolkits/mindone/tango/tango_full_ft_audiocaps-fa8f707f.ckpt). Ref: [tango-full-ft-audiocaps](https://huggingface.co/declare-lab/tango-full-ft-audiocaps).
```
[the remark this answers is not in the document]
Comment (genshimamber) [reply examples/tango/README.md:26]: done

**Review by SamitHuang — APPROVED** [review]
lgtm

**Pull request merged**